### PR TITLE
DDF-1097: Generalized pom file to generate all documentation with insert...

### PR DESCRIPTION
**Pull request by ricklarsen**
...ed version number.

### CODE REVIEW

File: security/docs/src/main/doc/IntegratingSecurity.adoc [hunk 1]
```diff
@@ -1,5 +1,5 @@
 = Integrating {branding} Security Application
-include::../includes/config.adoc[]
+include:: ../../classes/config.adoc[]
```
Comment (pklinef) [inline security/docs/src/main/doc/IntegratingSecurity.adoc]: I wish we could find a way to do this without using relative paths.

Comment (pklinef) [inline security/docs/src/main/doc/IntegratingSecurity.adoc]: http://asciidoctor.org/docs/user-manual/#include-files-relative-to-a-common-source-directory

Did you try that option?

Comment (ricklarsen) [inline security/docs/src/main/doc/IntegratingSecurity.adoc]: trying it now

**Comment by kcwire** [comment]
@ricklarsen can you update your PR to only include your commits?

**Comment by shaundmorris** [comment]
@kcwire looks like this only has @ricklarsen's commits now. Any other issues with this PR?

**Pull request merged**